src/Util.ts

```diff
@@ -63,14 +63,26 @@ export default class Util {
     const result = await fetch(url, init);
     return await result.json();
   }
-
+  static formatter(obj, str) {
+    return str.replace(/\$\{(.*?)\}/g, (match, variable) => {
+      if (obj.hasOwnProperty(variable)) {
+        const value = obj[variable];
+        if (typeof value === 'string') {
+          return value.replace(/"/g, '');
+        } else {
+          return value;
+        }
+      } else {
+        return match;
+      }
+    });
+  }
   static formatString(
     pattern: string,
     params: IDictionary<string> | any
   ): string {
-    const paraNameList = [...Object.getOwnPropertyNames(params)];
-    const formatter = new Function(...paraNameList, `return \`${pattern}\``);
-    return formatter(...paraNameList.map((x) => Reflect.get(params, x)));
```

Comment:
Please let me know why these lines of code are removed. The `formatter()` function has no similar functionality.